cuda/test/solver/lower_trs_kernels.cpp

```diff
@@ -133,12 +133,14 @@ TEST_F(LowerTrs, CudaLowerTrsFlagCheckIsCorrect)
 }
 
 
-TEST_F(LowerTrs, CudaSingleRhsApplyClassicalIsEquivalentToRef)
+TEST_F(LowerTrs, CudaSingleRhsApplySparselibIsEquivalentToRef)
```

Comment:
I think the path of classical is different from the sparselib

Reply:
Yeah, now it should be the default (sparselib) and syncfree